src/library/autodj/dlgautodj.cpp

```diff
@@ -170,6 +180,17 @@ DlgAutoDJ::DlgAutoDJ(
             this,
             &DlgAutoDJ::slotTransitionModeChanged);
 
+    faderModeCombobox->addItem(tr("Crossfader"),
```

Comment:
I don't think this will be set on the fly.
Did you consider to move this to the auto DJ preferences?

Reply:
No, I didn't consider that at first but now that you mention it it makes more sense to do so. Just to be sure I understand you, you are suggesting the option to switch between the crossfader and volume-faders should be in the auto DJ preferences, right?

Reply:
Yes.

Reply:
Done. It is now being set in auto DJ preferences: 2360eff041e876b2f17080991b3b2eb9c95f0994